libs/coin-modules/coin-solana/src/prepareTransaction.ts

```diff
@@ -439,6 +440,9 @@ async function deriveStakeCreateAccountCommandDescriptor(
   const errors: Record<string, Error> = {};
   const warnings: Record<string, Error> = {};
 
+  const commandDescriptor = tx.model.commandDescriptor;
+  if (isValidCommandDescriptor(commandDescriptor)) return commandDescriptor!;
```

Comment:
```suggestion
  if (isStakeCreateAccountCommandDescriptor(commandDescriptor)) return commandDescriptor;
```

Note we don't need to use `!` anymore now that we are actually using a predicate

Reply:
https://github.com/LedgerHQ/ledger-live/pull/9602/commits/a956a96b18b9f5614948027848f05de467f94164